examples/metrics/coco/recall_demo.py

```diff
@@ -0,0 +1,26 @@
+import tensorflow as tf
```

Comment:
I guess we can skip this demo, since it is included in the docstring, or if it is not, then we should put it in the docstring.

Reply:
Yeah, I can remove the demo.

Reply:
Done